web/ViewModels/LandlordListModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿using BricksAndHearts.Database;
+namespace BricksAndHearts.ViewModels;
+
+public class LandlordListModel
+{
+    public List<LandlordDbModel>? UnapprovedLandlords { get; set; }

```

Comment:
Does this need to be nullable? If we don't expect this value to ever be null then it would be best to make it non-nullable 🙂

Reply:
That sounds reasonable 😄 are we sure that we get a null and not an empty list in that scenario?

Reply:
I'm not certain, but if I don't set it as nullable it causes a build warning which stops me from raising a pull request.